web/src/lib/tax-access/taxAccess.ts

```diff
@@ -0,0 +1,83 @@
+import { postTaxFilingsOnboarding } from "@/lib/api-client/apiClient";
+import { UpdateQueue } from "@/lib/types/types";
+import {
+  Business,
+  getCurrentBusiness,
+  LookupLegalStructureById,
+  ProfileData,
+} from "@businessnjgovnavigator/shared";
+import { Dispatch, SetStateAction } from "react";
+
+interface TaxAccessProps {
+  profileData: ProfileData;
+  business?: Business;
+  setIsLoading?: Dispatch<SetStateAction<boolean>>;
+  updateQueue?: UpdateQueue;
+}
+
+export const displayBusinessName = (business?: Business): boolean => {
+  return LookupLegalStructureById(business?.profileData.legalStructureId).elementsToDisplay.has(
+    "businessName"
+  );
+};
+
+export const displayResponsibleOwnerName = (business?: Business): boolean => {
+  return LookupLegalStructureById(business?.profileData.legalStructureId).elementsToDisplay.has(
+    "responsibleOwnerName"
+  );
+};
+
+export const taxAccess = async ({
```

Comment:
dust - think you meant to use this code in the SidebarCardFunding nudge? I'm not sure where this is used at the moment as I did a global search and didn't find anything but it could be my IDE